e2e/images_test.go

```diff
@@ -23,7 +24,11 @@ func insertBundles(t *testing.T, cmd icmd.Cmd) {
 
 func assertImageListOutput(t *testing.T, cmd icmd.Cmd, expected string) {
 	result := icmd.RunCmd(cmd).Assert(t, icmd.Success)
-	match, _ := regexp.MatchString(expected, result.Stdout())
+	stdout := result.Stdout()
+	match, _ := regexp.MatchString(expected, stdout)
+	if !match {
+		fmt.Println(stdout)
+	}
 	assert.Assert(t, match)
```

Comment:
I think you can add a third argument here, instead of checking `match` before assertion.
`assert.Assert(t, match, stdout)`